### DISCUSSION TITLE
Error before using WordPress example

### DISCUSSION THREAD

**Discussion by ahmedsamirdev** [vercel/next.js#27905] [Help]
I downloaded the example from [here](https://github.com/vercel/next.js/tree/canary/examples/cms-wordpress),
set my wp_url WORDPRESS_API_URL=http://www.*********/graphql
and giving me the graphql normal error msg but when try to initiate (npm run dev),
I receive this error ..
`FetchError: invalid json response body at http://www.*********/graphql reason: Unexpected token < in JSON at position 0`

**Comment by iorippi** [comment]
What is your response body? I had a similar case where the `<` character was the response body being html file (thus the response starts with `<DOCTYPE..`) because the url for api was not set properly and it was returning html error page to the request.

WPGraphQL plugin's General Settings page might say "GraphQL Endpoint: graphql", but you should be able to see the full url for the api right beneath the field. For me, it was `example.com/wp/graphql`, not `example.com/graphql`.

**Comment by gajen007** [comment]
In your code, comment the .json( ) method and try with .text( ) method as the return can be seen in console.log( ). Then you may find that HTML code which shows the actual error.
